server/routes/wazuh-api.ts

```diff
@@ -153,4 +153,13 @@ export function WazuhApiRoutes(router: IRouter) {
   },
     async (context, request, response) => ctrl.isWazuhDisabled(context, request, response)
   );
+
+  // Return app logos configuration
+  router.get({
+    path: '/api/get-logos',
```

Comment:
No need to add the action (HTTP verb) to the URL, it's already defined in the route (get).

It's not a good practice to do this, check guideline nr.1 from this [Best Practices for naming REST API endpoints](https://betterprogramming.pub/10-best-practices-for-naming-rest-api-endpoints-442ae592a3a0) guide.

Reply:
As @asteriscos is on holidays, I'll apply this change.

Reply:
Solved in https://github.com/wazuh/wazuh-kibana-app/pull/4539/commits/0ace60f0912eb2fb9363999ba9180ffc135486f0